guides/release/tutorial/13.md

```diff
@@ -0,0 +1,93 @@
+
+This way, our `<Rental::Image>` component is not coupled to any specific rental property on the site. Of course, the problem still exists, we simply moved it to the `<Rental>` component, but we will deal with that soon. We can try to limit all the hard-coding to the `<Rental>` component, so that we will have an easier time cleaning it up when we switch to fetching real data.
+
+In general, it is a good idea to add `...attributes` to the primary element in your component. This will allow for maximum flexibility as the invoker may need to pass along classes for styling, or ARIA attributes to improve accessibility.
+
+Let's write a test for our component!
+
+<!-- TODO: format diff -->
+```
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+ module('Integration | Component | rental/image', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders the given image', async function(assert) {
+    await render(hbs`
+      <Rental::Image
+        src="/assets/images/teaching-tomster.png"
+        alt="Teaching Tomster"
+      />
+    `);
+
+    assert.dom('.image').exists();
+    assert.dom('.image img').hasAttribute('src', '/assets/images/teaching-tomster.png');
+    assert.dom('.image img').hasAttribute('alt', 'Teaching Tomster');
+  });
+});
+```
+
+Finally, we should also update the tests for the `<Rental>` component to confirm that we successfully invoked `<Rental::Image>`.
+
+<!-- TODO: format diff -->
+```
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | rental', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders information about a rental property', async function(assert) {
+    await render(hbs`<Rental />`);
+
+    assert.dom('article').hasClass('rental');
+    assert.dom('article h3').containsText('Grand Old Mansion');
+    assert.dom('article .detail.owner').containsText('Veruca Salt');
+    assert.dom('article .detail.type').containsText('Standalone');
+    assert.dom('article .detail.location').containsText('San Francisco');
+    assert.dom('article .detail.bedrooms').containsText('15');
+    assert.dom('article .image').exists();
+  });
+});
+```
+
+Because we already tested `<Rental::Image>` extensively on its own, we can omit the details here and keep our assertion to the bare minimum. That way, we won't have to _also_ update the `<Rental>` tests whenever we make any changes to `<Rental::Image>`.
```

Comment:
```suggestion
Because we already tested `<Rental::Image>` extensively on its own, we can omit the details here and keep our assertion to the bare minimum. That way, we won't  _also_ have to update the `<Rental>` tests whenever we make changes to `<Rental::Image>`.
```